src/styles/_sweet-alert-overrides.scss

```diff
@@ -12,6 +12,10 @@
 
 }
 
+.swal-wide {
```

Comment:
What is this for?

Reply:
This was for other issue related to #872 
As I remember, I create each branch from develop. I don't know why this line exists in scss. Howere, this should not be applied in this commit. Thanks @arbreton .